pyproject.toml

```diff
@@ -69,7 +69,7 @@ Source = "https://github.com/iterative/dvc-s3"
 
 [tool.pytest.ini_options]
 log_level = "debug"
-addopts = "-ra"
+addopts = "-ra -pdvc.testing.plugin"
```

Comment:
We cannot keep this inside `conftest.py`. 
```python
pytest_plugins = ["dvc.testing.plugin"]
```

`pytest` only supports them if they are at the top-level, eg: `tests/conftest.py`. But the tests package in dvc-s3 is inside `dvc_s3/tests`.